src/gpuarray_reduction.c

```diff
@@ -726,10 +810,11 @@ static int   invokeMaxAndArgmax     (GpuKernel*       k,
 	 * 
 	 * This implies choosing the block, grid and chunk size appropriately.
 	 */
-	
+	/*
 	scheduleMaxAndArgmax(k, src, reduxLen, reduxList,
 	                     blockSize, gridSize, chunkSize);
-	
+	*/
+	(void)scheduleMaxAndArgmax;
```

Comment:
@abergeron As I wrote in my email, I was doing static scheduling to make sure the rest works. You can also see a hardcoded `50` above. This will disappear soon, I'm working on it.